### ISSUE TITLE
Cleanup/deprecation

### ISSUE THREAD

**Issue by jklmnn** [Componolit/ada-runtime#86]
With the update to GNAT 12 I noticed that there are more and more files that change in newer GNAT versions. With the latest version the pragma `Compiler_Unit_Warning` is no longer available which initially made the build fail on multiple occasions. I quick fixed that now by ignoring unknown pragmas with `-gnatwG`.

While trying to fix this I noticed that we have no proper solution for different versions. We have several copies of `a-tags.ads`, one for each compiler. All but one of them are the exact same file.

I see two options now:
 - Fix the build system to handle these things better (and move the runtime to alire in that process)
 - Discontinue the runtime

Currently I tend to favor the second option. Apart from posix none of the other platforms has been properly tested in years and as far as I can tell nobody is using this runtime.

@treiher @senier what's your opinion on this?

**Comment by senier** [comment]
@jklmnn I agree with you - we should deprecate it. It's not actively used and seems to become a maintenance burden with no benefit.
